core/templates/cowdata.h

```diff
@@ -342,7 +343,7 @@ Error CowData<T>::resize(Size p_size) {
 	}
 
 	// possibly changing size, copy on write
-	USize rc = _copy_on_write();
```

Comment:
Was rc just unused?

Reply:
It used to be used as the initial value of the new refcount. In this case, it was guaranteed to be 1, because copy_on_write always returns 1 (unless ptr is null, in which case it returns 0). 
And yeah, it makes perfect sense that it's always 1 after a fork because that's what forking guarantees, lol.

Reply:
Right, and if we try to resize to 0, where _copy_on_write would return 0, we never make it to this method. And indeed in all other cases, if we *do* copy, we always get back an rc of 1.